include/Game/Gravity/PointGravity.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "Game/Gravity/PlanetGravity.h"
+
+class PointGravity : public PlanetGravity {
+public:
+	PointGravity();
+
+	virtual void updateMtx(const TPos3f &rMtx);
+	virtual bool calcOwnGravityVector(TVec3f *pDest, f32 *pScalar, const TVec3f &rPosition) const;
+
+	TVec3f mOrigPosition; // _28
```

Comment:
not sure if using two variations of the same name (ie Position and Translation) can work here, perhaps use one or the other